examples/shader/blackhole.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build ignore
+// +build ignore
+
+package main
+
+var Time float
+var ScreenSize vec2
+var Cursor vec2
+
+func Fragment(position vec4, texCoord vec2, color vec4) vec4 {
+	origin, size := imageDstRegionOnTexture()
+	texC := (position.xy/imageDstTextureSize() - origin) / size
+	texC2 := (position.xy/imageDstTextureSize().x - origin) / size.x
```

Comment:
I'm not sure what this means. Why is the position divided by the width?

Reply:
I am not sure either, it was adjusted from this shader toy example https://www.shadertoy.com/view/Xt3GWj

Reply:
Please make the behavior clear to you before sending this PR... Thanks!

Reply:
It is divided by the longest side (so the width usually on screens) so that the circle doesn't appear streched on X axis

Maybe we should try running this example with a resolution that has a height > width ?
That would mean we should divide it by the max of both